tests/Unit/Domain/ValueObject/GeographicPositionTest.php

```diff
@@ -12,26 +12,26 @@
 namespace Eluceo\iCal\Unit\Domain\ValueObject;
 
 use Eluceo\iCal\Domain\ValueObject\GeographicPosition;
-use Generator;
 use InvalidArgumentException;
+use PHPUnit\Framework\Attributes\DataProvider;
 use PHPUnit\Framework\TestCase;
 
 class GeographicPositionTest extends TestCase
 {
-    /**
-     * @dataProvider provideInvalidPositions
-     */
+    #[DataProvider('provideInvalidPositions')]
     public function testConstructorDoesNotAcceptInvalidArguments(float $latitude, float $longitude)
     {
         static::expectException(InvalidArgumentException::class);
         new GeographicPosition($latitude, $longitude);
     }
 
-    public static function provideInvalidPositions(): Generator
+    public static function provideInvalidPositions(): array
     {
-        yield [-91, 0];
-        yield [91, 0];
-        yield [0, -181];
-        yield [0, 181];
```

Comment:
To be honest: I never knew generators work for dataproviders, but it's not how PHPUnit recommend them to be used.
Therefore I changed it to the recommended array syntax

Reply:
PHPUnit doesn't recommend a specific syntax, and it explicitly supports generators. Which you want to use is a developer choice.

So, I would revert those changes to keep the amount of changes as low as possible.

Reply:
Well this library seems pretty dead anyways so I'm gonna fork it and use my own fork